### PR TITLE
Update codecov.yml

### LINKED ISSUE #36: Restrict target files of measuring test coverage
## Description

Limit test coverage measurement to specific modules in src/

### Ignore

- main.c
- ui
- utils

### PR DESCRIPTION
**Pull request by rask24**
fix #36

## Summary by CodeRabbit

- **Chores**
	- Updated coverage reporting configuration to exclude additional files and directories, enhancing clarity and focus of coverage metrics.

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -19,8 +19,11 @@ coverage:
         base: auto
 
 ignore:
-  - "main.c"
-  - "tests/*"
+  - "tests/**/*"
+  - "src/main.c"
+  - "src/ui/*"
+  - "src/utils/*"
```
Comment (alex216) [inline codecov.yml:25]: is it sufficient to add only `src/utils/**/*`?

**Review by alex216 — CHANGES_REQUESTED** [review]
check the comment @rask24

File: codecov.yml (hunk 1 above)
Comment (rask24) [reply codecov.yml:25]: In `f4caab081ccc286882b8a807f92d88da19a45bd4`,
I added `src/ui/**/*` and `src/utils/**/*` BUT codedov ignored ONLY files in `src/utils/wrapper`.
I think `src/utils/*` indecates files in `src/utils` and `src/utils/**/*` indecates files in "subdirectories" of `src/utils`.

**Comment by rask24** [comment]
I tried to delete `src/utils/**/*`.

**Review by alex216 — APPROVED** [review]
good to go. @rask24

**Pull request merged**